pghistory/models.py

```diff
@@ -129,7 +132,13 @@ def get_compiler(self, *args, **kwargs):
         return compiler
 
 
-class EventQuerySet(models.QuerySet):
+if TYPE_CHECKING:
+    _EventQuerySetBase = models.QuerySet[_M]
+else:
+    _EventQuerySetBase = models.QuerySet
```

Comment:
Want to make sure users who don't have `QuerySet` patched as a generic don't run into issues.